cache/binary/tags.go

```diff
@@ -124,6 +124,13 @@ func init() {
 	addCommonKey("addr:postcode")
 	addCommonKey("addr:housenumber")
 
+	addCommonKey("osm_changeset")
```

Comment:
I have added  ->  "const ParseMetadataPrefix = "osm_"  
because:
* I have lot of osm2pgsql code,  so compatibility is important for me.
* to prevent collisions with similar osm keys : 
 * http://taginfo.openstreetmap.org/keys/changeset  (  count :  19 )
 * http://taginfo.openstreetmap.org/keys/version  ( count : 3448 )
 * http://taginfo.openstreetmap.org/keys/user ( count  : 197 )